src/rgw/rgw_sync.cc

```diff
@@ -1084,9 +1084,10 @@ class RGWAsyncMetaStoreEntry : public RGWAsyncRadosRequest {
   rgw::sal::RGWRadosStore *store;
   string raw_key;
   bufferlist bl;
+  const DoutPrefixProvider *dpp;
```

Comment:
@cbodley the two constructors that takes dpp's in this file are only used in this file and take a sync_env->dpp. Are they fine?